pyEpiabm/pyEpiabm/tests/test_covidsim_helpers.py

```diff
@@ -1,36 +1,57 @@
 import unittest
 import pyEpiabm as pe
+from pyEpiabm import CovidsimHelpers as c
 
 
 class TestCovidsimHelpers(unittest.TestCase):
-    """Test the 'CovidsimHelpers' class.
+    """Test the 'CovidsimHelpers' class, which contains the
+    infectiousness and susceptibility calculations that
+    determine whether infection events occur. Each function
+    should return a number greater than 0.
     """
-
     @classmethod
     def setUpClass(cls) -> None:
+        """Intialise a population with one infector and one
+        infectee, both in the same place and household.
+        """
         cls.cell = pe.Cell()
         cls.microcell = pe.Microcell(cls.cell)
         cls.infector = pe.Person(cls.microcell)
         cls.infectee = pe.Person(cls.microcell)
+        cls.place = pe.Place((1, 1), pe.PlaceType.Hotel, cls.cell,
+                             cls.microcell)
+        cls.place.add_person(cls.infector)
+        cls.place.add_person(cls.infectee)
+        cls.timestep = 1
 
     def test_calc_house_inf(self):
-        value = pe.CovidsimHelpers.calc_house_inf(self.infector, 0)
-        self.assertEqual(value, 1.0)  # Will have to update with function
-        self.assertIsInstance(value, float)
+        result = c.calc_house_inf(self.infector, self.timestep)
+        self.assertTrue(result > 0)
+        self.assertIsInstance(result, (float, int))
```

Comment:
```suggestion
        self.assertIsInstance(result, float)
```

Documentation says it will return a float so I think best to assume this is the case, and convert an int outputs to float if necessary

Reply:
(Also same comment to further uses of this construction in this file)